tree/dataframe/test/dataframe_friends.cxx

```diff
@@ -63,9 +63,9 @@ TEST_F(RDFAndFriends, FriendByFile)
 TEST_F(RDFAndFriends, FriendByPointer)
 {
    TFile f1(kFile1);
-   TTree *t1 = static_cast<TTree *>(f1.Get("t"));
+   auto t1 = f1.Get<TTree>("t");
```

Comment:
To improve the stability of the test you could consider testing f1 for IsZombie (missing file) and check the value of t1 (wrong file).

Reply:
At this point I fully expect that the file exists and is well formed. It's the job of the test fixture to guarantee this is the case.

Reply:
So it is probably not worth to upgrade those test but in general it would be better (and even in this test many thing could go wrong [out of space disk, new bug in file creation, etc...]

Reply:
I think this is a feature request for `RDataFrame::Snapshot`: it should complain noisily if something goes wrong in the creation of the output file. Feel free to open a jira ticket!

Reply:
the 2 operations here can fail (other cases could be disk failure that happened between the writing and reading, future mistaken by updater that change the name of the ttree in one part but not the other, etc .... i.e. even success of the writing is no guarantee of the success of the reading).   It is just best to add (in particular in a 'test' :) ) those kind of tests (when practical and easy).   [again this is not 'important' enough (as it just will 'just'/'only' make debugging problem easier for the few rare-ish cases i mentioned) to warrant a PR for this particular test but it is worth getting in the habit of doing (and yes, many, too many, of the existing tests are not up to notch in that regards)]